bundle/tests/scorecard/kuttl/test-db-pvc-tee/01-pods.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: test-db-pvc-tee
+spec:
+  targetNamespace: test-db-pvc-tee
+  providers:
+    web:
+      port: 8000
+      mode: operator
+    metrics:
+      port: 9000
+      mode: operator
+      path: "/metrics"
+    kafka:
+      mode: none
+    db:
+      mode: local
+      pvc: true
+      tShirtSizeDefinitions:
+        small: 2Gi
+        medium: 3Gi
+        large: 4Gi
+    logging:
+      mode: none
+    objectStore:
+      mode: none
+    inMemoryDb:
+      mode: none
+    featureFlags:
+      mode: none
+  resourceDefaults:
+    limits:
+      cpu: 400m
+      memory: 1024Mi
+    requests:
+      cpu: 30m
+      memory: 512Mi
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdApp
+metadata:
+  name: app-a
+  namespace: test-db-pvc-tee
+spec:
+  envName: test-db-pvc-tee
+  deployments:
+  - name: processor
+    podSpec:
+      image: quay.io/psav/clowder-hello
+  database:
+    name: app-a
+    version: 12
+    pvc: true
```

Comment:
Do we want to add any testing around setting `pvc: false` and trying to have a size enabled? We'll need to update the provider docs as well. I don't have a strong opinion on TShirtSize vs DBSize or VolumeSize. Looks good otherwise!